app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # Include default devise modules.
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+         :recoverable, :rememberable, :trackable,
+         :validatable, :omniauthable
+  include DeviseTokenAuth::Concerns::User
```

Comment:
While it's not a rule, we tend to have the includes right at the top of the class, after the `class User < ApplicationRecord`, mostly because the includes add code to a class and it's harder to track otherwise